ophyd/device.py

```diff
@@ -1274,7 +1274,7 @@ def remove(self, value):
             getattr(self._parent, value).kind &= ~self._remove_kind
 
         def __contains__(self, value):
-            return getattr(self._parent, value).kind & self._kind
+            return self.__internal_list().__contains__(value)
```

Comment:
Might as well write this as `value in self.__internal_list()`

Reply:
Probably correct for readability, I'll swap it